graphql/schema/gqlschema.go

```diff
@@ -2316,6 +2330,24 @@ func getFieldsWithoutIDType(schema *ast.Schema, defn *ast.Definition, providesTy
 	return append(fldList, pd)
 }
 
+// This function check if given gql field has multiple language tags
+func isMultiLangField(fld *ast.FieldDefinition, isMutationInput bool) bool {
+	dgDirective := fld.Directives.ForName(dgraphDirective)
+	if dgDirective != nil {
+		pred := dgDirective.Arguments.ForName("pred")
+		if pred != nil {
+			if strings.Contains(pred.Value.Raw, "@") {
+				langs := strings.Split(pred.Value.Raw, "@")[1]
```

Comment:
what if there is nothing after @. We should use indexing than using strings.Split here